.github/workflows/build-exporter-image.yaml

```diff
@@ -0,0 +1,53 @@
+name: Building kube-events-exporter image
+
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+    paths:
+      - ".github/workflows/build-exporter-image.yaml"
+      - "cmd/operator/**"
+      - "hack/**"
+      - "pkg/**"
+      - "Makefile"
+
+env:
+  REPO_EXPORTER: 'kubesphere'
+
+jobs:
+  exporter-build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    name: Build Image for kube-events-exporter
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.x
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and Push Image for kube-events-exporter
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          make cross-build-exporter -e REPO_EXPORTER=${{ env.REPO_EXPORTER }}/kube-events-exporter
```

Comment:
` docker run --rm --privileged multiarch/qemu-user-static --reset -p yes` would be useful for custom building, I think it integrating it in the Makefile is suitable.

Reply:
> or use this action [docker/setup-qemu-action](https://github.com/docker/setup-qemu-action) instead

Yeah. I am confusing about that some image like fluentd perhaps not supported this action, refer to https://github.com/fluent/fluentd-docker-image/blob/master/v0.14/debian-onbuild/hooks/post_push.  Maybe we need a test for it. For common speaking, this action would support.